Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
This check worries me a bit. I think you're assuming that the master branch is always from eslint/eslint, but that's not a safe assumption. Someone could have forked the repo and created an upstream branch to track eslint/master.

So right now, this would really only with for members of the team and not anyone else.